manifests/z/Zeal8bit/Zeal-WebEmulator/0.1.0/Zeal8bit.Zeal-WebEmulator.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
```

Comment:
Are `manifest` files required? Are they automatically generated?

Reply:
No, It needs [`wingetcreate`](https://github.com/microsoft/winget-create), maby you can use wine to use it on linux😔

Reply:
Can you remove them for now?